### ISSUE TITLE
Bugs in igmix

### ISSUE THREAD

**Issue by Agusmastex** [chmarti1/PYroMat#28]
When passing density as a parameter in the functions for enthalpy, entropy and internal energy
```
air = pm.get('ig.air')  
pm.config['unit_temperature'] = 'C'  
pm.config['unit_pressure'] = 'kPa'
d0 = air.d(T=298.15, p=101.325)
air.s(T=300,d=d0)
```

The following error is raised.
```
`KeyError                                  Traceback (most recent call last)
<ipython-input-37-ffa8537a3566> in <module>
      3 pm.config['unit_pressure'] = 'kPa'
      4 d0 = air.d(T=298.15, p=101.325)
----> 5 air.s(T=300,d=d0)

c:\python38\lib\site-packages\pyromat\registry\igmix.py in s(self, *varg, **kwarg)
    678 """
    679         self._bootstrap()
--> 680         T,p = self._argparse(*varg, temperature=True, pressure=True, **kwarg)
    681         s = self._s(T)[0] - pm.units.const_Ru * np.log(p/(1e5*self._pref_bar))
    682         pm.units.energy(s, from_units='kJ', inplace=True)

c:\python38\lib\site-packages\pyromat\registry\igmix.py in _argparse(self, T, p, d, temperature, pressure, density)
    209 
    210         if d is not None:
--> 211             d = pm.units.matter(np.asarray(d,dtype=float), self.data['mw'], to_units='kmol')
    212             pm.units.volume(d, to_units='m3', exponent=-1, inplace=True)
    213             if d.ndim==0:

KeyError: 'mw'
```

When trying to use igmix.gam, igmix.cp or igmix.cv: 
```
air.s(T=300,d=d0)

AttributeError                            Traceback (most recent call last)
<ipython-input-60-6447cf26e838> in <module>
      2 pm.config['unit_temperature'] = 'C'
      3 pm.config['unit_pressure'] = 'kPa'
----> 4 d0 = air.cp(T=298.15, p=101.325)
      5 air.s(T=300,d=d0)

c:\python38\lib\site-packages\pyromat\registry\igmix.py in cp(self, *varg, **kwarg)
    544 Returns:    Spec. Heat  [unit_energy / unit_temperature / unit_matter]
    545 """
--> 546         self._boostrap()
    547         # Parse the arguments to isolate temperature in K
    548         T = self._argparse(*varg, temperature=True, **kwarg)

AttributeError: 'igmix' object has no attribute '_boostrap'
```

**Comment by chmarti1** [comment]
The bug is corrected in release v2.1.10.  To patch,
```bash
$ python3 -m pip install --upgrade pyromat
```

**Comment by chmarti1** [comment]
** Note **
The second bug noted in the edited post was a keystroke error corrected in v2.1.4

**Comment by Agusmastex** [comment]
Thank you so much!
